lib/uwb/UwbMacAddress.cxx

```diff
@@ -39,6 +41,36 @@ UwbMacAddress::GetValueShort() const
     return (static_cast<uint16_t>(m_view[1]) << 8U) | m_view[0];
 }
 
+UwbMacAddress::UwbMacAddress(const std::string& addressString, UwbMacAddressType addressType)
```

Comment:
I was unable to make this one a constexpr like the other constructors due to std::stringstream. Perhaps there is a way around it that I don't know about?